### DISCUSSION TITLE
How to hot reload styles (For lib developing)?

### DISCUSSION THREAD

**Discussion by etamity** [react-page/react-page#897] [Q&A]
Hi Team, 
I cloned the repo and ran `yarn && yarn dev`, try to change css , but it seems not able to pick up changes, do I miss any steps?

Thanks

**Comment by macrozone** [comment]
> Hi Team,
> I cloned the repo and ran `yarn && yarn dev`, try to change css , but it seems not able to pick up changes, do I miss any steps?
> 
> Thanks

css are indeed not watched, could not find a fix yet. But its using postcss and i'll guess there is some watch mode for that

**Reply by etamity** [reply]
@macrozone Thanks, i wonder how you guys develop the library without watching css though.

**Reply by macrozone** [reply]
i did not change that much on the css ;-)

to be honest: it annoyed me as well, but i did not want to go into it yet. I am always glad if people help improving the tooling, its a nessecary evil.
